### ISSUE TITLE
Things to add in README

### ISSUE THREAD

**Issue by aaaaalbert** [IndoorLocalization/datasets#1]
Here are a few things we should add to the README. I'm sure there are more.
- Explain how, where, by whom the data was recorded
- Add floor plan
- Describe the structures of the different file formats
- Describe the units of measurement for values (microseconds since when? cm? uT? ...), and how the QR codes map into (x,y)
- Explain why not all data points have values for all dimensions

**Comment by yyzhuang** [comment]
Except the floor plan (we only use QR codes to checkin), I agree with Albert's comments.

**Comment by aaaaalbert** [comment]
Right, but we still need a map of where in the room the QR codes were, and what places could not be visited physically due to the presence of walls. Adds info about ground truth, right?
